components/lang/src/main/java/io/cellery/CelleryConstants.java

```diff
@@ -32,4 +32,8 @@
     public static final String CELL_REFERENCE_TEMPLATE_CONTEXT_COMPONENTS = "components";
     public static final String CELL_REFERENCE_TEMPLATE_CONTEXT_HANDLE_API_NAME = "handleApiName";
     public static final String CELL_REFERENCE_TEMPLATE_CONTEXT_HANDLE_TYPE_NAME = "handleTypeName";
+
+    public static final String TARGET = "target";
+    public static final String RESOURCES = "resources";
+    public static final String GATEWAY_PROTOCOL = "http";
```

Comment:
As the gateway protocol can be different transports, any reason for this to stick to HTTP? We can probably rename this to HTTP_GATEWAY_PROTOCOL as the constant name, or if it's default protocol, let's change this to DEFAULT_GATEWAY_PROTOCOL

Reply:
This is the default gateway protocol. Changed it to DEFAULT_GATEWAY_PROTOCOL. (https://github.com/cellery-io/sdk/pull/25/commits/427234b689041403a2033292514d323df5267697)